main.py

```diff
@@ -372,7 +372,10 @@ def user_token(self, userid='core'):
     def check_docker(self):
         '''Check docker services status'''
         stack = 'docker-cloud stack inspect %s' % self.stackname
-        self._run(stack)
```

Comment:
@zelima  since `self._run` function prints entire response. if  I need only state part, then I have to change `_run` function, but i did not want to touch it since it is used for other tests